release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -288,6 +313,11 @@ indicate that the feature is removed from the release or deprecated.
 |TP
 |TP
 
+|iSCSI Persistent Volumes
+|
```

Comment:
If it didn't apply for 4.1, looks like you should add "-" like the rest of the table does for N/A items.

Reply:
I actually had it in before noticing that rows at the end left it out. Put it back in. :)